### ISSUE TITLE
[ERR_UNSUPPORTED_ESM_URL_SCHEME]: Only file and data URLs are supported by  the default ESM loader.

### ISSUE THREAD

**Issue by simonnilsson** [i18next/i18next-parser#527]
## 🐛 Bug Report

Latest version (6.0.0) does not run on Windows. It fails while trying to open config file. Doesn't matter if you supply a custom path or it just tries to load default **i18next-parser.config.js**. 

The following error is given while running the basic **i18next** command:

```
Error [ERR_UNSUPPORTED_ESM_URL_SCHEME]: Only file and data URLs are supported by 
the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'c:'
    at Loader.defaultResolve [as _resolve] (internal/modules/esm/resolve.js:777:11)
    at Loader.resolve (internal/modules/esm/loader.js:88:40)
    at Loader.getModuleJob (internal/modules/esm/loader.js:241:28)
    at Loader.import (internal/modules/esm/loader.js:176:28)
    at importModuleDynamically (internal/modules/esm/translators.js:114:35)
    at exports.importModuleDynamicallyCallback (internal/process/esm_loader.js:30:14)
    at file:///****/node_modules/i18next-parser/bin/cli.js:51:33
```

Where `c:` is the drive the project is in.

## To Reproduce

```sh
npm init
npm i i18next-parser
npx i18next
```

## Expected behavior

Command to run without errors.

## Environment

- *runtime version*: node v16.14.2
- *i18next-parser version*: 6.0.0
- *os*: Windows 10

**Comment by simonnilsson** [comment]
Changing

https://github.com/i18next/i18next-parser/blob/c2dccd51cf6a1a6de8fb78cfa3f87bd03e11d005/bin/cli.js#L51

to

```js
import { pathToFileURL } from 'url';
config = (await import(pathToFileURL(path.resolve(program.opts().config)))).default
```

fixes the issue. I can make a pull request for this change if wanted.

**Comment by karellm** [comment]
This should be fixed in `6.0.1`. Please note that this version dropped support for Node 12 also.
